rdr_service/genomic/genomic_job_components.py

```diff
@@ -548,6 +550,13 @@ def _ingest_aw4_manifest(self, rows):
 
                 self.member_dao.update(member)
 
+                dispatch_task(
```

Comment:
The AW4 job runs when either an Array or a WGS AW4 comes through. The AW4 itself will be for one of these two genome types. Do you want to update the enrollment status on either the Array or WGS, or do you only want to update it if the WGS sample passes QA. If the latter is the case, you'll want to make sure the `self.job_id == GenomicJob.AW4_WGS_WORKFLOW`

Reply:
My thinking was that I didn't want to put duplicate business logic here for when the task should be fired, but instead have the event triggered and the calculation code determine what it reads for the calculation. That way, if something changes later, it's simple enough to remember that an AW4 triggers the calculation and the calculation is the only thing that needs to change. Rather than needing to remember that the AW4 only triggers the calculation if it's a WGS, and then both the calculation code and the triggering code would need to be updated.

But I know that's debatable, so if you want it changed then I can go ahead and make the update.